dev/io.openliberty.jakarta.cdi.4.0/bnd.bnd

```diff
@@ -11,38 +11,44 @@
- jakarta.enterprise.event; version=3.0, \
- jakarta.enterprise.inject; version=3.0, \
- jakarta.enterprise.inject.literal; version=3.0, \
- jakarta.enterprise.inject.se; version=3.0, \
- jakarta.enterprise.inject.spi; version=3.0, \
- jakarta.enterprise.inject.spi.configurator; version=3.0, \
- jakarta.enterprise.util; version=3.0, \
- jakarta.inject; version=2.0.1
-
+Export-Package: \
+ jakarta.decorator;version="3.0";uses:="jakarta.enterprise.inject",\
+ jakarta.enterprise.context;version="3.0";uses:="jakarta.enterprise.util,jakarta.inject",\
+ jakarta.enterprise.context.control;version="3.0";uses:="jakarta.enterprise.context,jakarta.interceptor",\
+ jakarta.enterprise.context.spi;version="3.0",\
+ jakarta.enterprise.event;version="3.0";uses:="jakarta.enterprise.util",\
+ jakarta.enterprise.inject;version="3.0";uses:="jakarta.enterprise.context,jakarta.enterprise.inject.spi,jakarta.enterprise.util,jakarta.inject",\
+ jakarta.enterprise.inject.build.compatible.spi;version="3.0";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject,jakarta.enterprise.inject.spi,jakarta.enterprise.lang.model,jakarta.enterprise.lang.model.declarations,jakarta.enterprise.lang.model.types",\
+ jakarta.enterprise.inject.literal;version="3.0";uses:="jakarta.enterprise.util,jakarta.inject",\
+ jakarta.enterprise.inject.se;version="3.0";uses:="jakarta.enterprise.inject,jakarta.enterprise.inject.spi",\
+ jakarta.enterprise.inject.spi;version="3.0";uses:="jakarta.el,jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject,jakarta.enterprise.inject.spi.configurator,jakarta.interceptor",\
+ jakarta.enterprise.inject.spi.configurator;version="3.0";uses:="jakarta.enterprise.context.spi,jakarta.enterprise.event,jakarta.enterprise.inject,jakarta.enterprise.inject.spi,jakarta.enterprise.util",\
+ jakarta.enterprise.util;version="3.0",\
+ jakarta.inject;version="2.0.1",\
+ jakarta.enterprise.lang.model;version="4.0";uses:="jakarta.enterprise.lang.model.declarations,jakarta.enterprise.lang.model.types",\
+ jakarta.enterprise.lang.model.declarations;version="4.0";uses:="jakarta.enterprise.lang.model,jakarta.enterprise.lang.model.types",\
+ jakarta.enterprise.lang.model.types;version="4.0";uses:="jakarta.enterprise.lang.model,jakarta.enterprise.lang.model.declarations"
 
-Import-Package:  jakarta.el; version="[4.0.0,5.0.0)", \
- jakarta.decorator; version=3.0, \
- jakarta.enterprise.context; version=3.0, \
- jakarta.enterprise.context.control; version=3.0, \
- jakarta.enterprise.context.spi; version=3.0, \
- jakarta.enterprise.event; version=3.0, \
- jakarta.enterprise.inject; version=3.0, \
- jakarta.enterprise.inject.literal; version=3.0, \
- jakarta.enterprise.inject.spi; version=3.0, \
- jakarta.enterprise.inject.spi.configurator; version=3.0, \
- jakarta.enterprise.util; version=3.0, \
- jakarta.inject; version=2.0, \
- jakarta.interceptor;version=2.0, \
+Import-Package: \
+ jakarta.el;version="4.0",\
```

Comment:
Shouldn't the import be a range?

Reply:
I don't know ... I copied most of these imports and exports from the original bundles. The original `jakarta.enterprise.cdi-api` manifest does not use a range for this so neither have I.